docker/config-validation/configs/workers/valid/odyssey-test3.conf

```diff
@@ -0,0 +1,7 @@
+workers "auto"
+
+log_format "%p %t %l [%i %s] (%c) %m\n"
+
+listen {
```

Comment:
maybe configs without any rules should not be valid, as they are not usable?

Reply:
done